my-app/src/app/models/course.ts

```diff
@@ -7,6 +7,7 @@ export class Course implements CourseInterface {
   public creationDate: Date;
   public duration: number;
   public description: string;
+  public topRated: boolean;
```

Comment:
should also be a property in courseList interface

Reply:
done